bb-aem-65/src/main/java/com/cognifide/qa/bb/aem/core/component/dialog/dialogfields/DefaultPathBrowser.java

```diff
@@ -52,8 +53,8 @@
   public void setValue(Object value) {
     input.clear();
     input.sendKeys(String.valueOf(value));
-
-    bobcatWait.until(elementToBeClickable(firstResult)).click();
+    bobcatWait.until(elementToBeClickable(firstResult));
+    input.sendKeys(Keys.ENTER);
```

Comment:
Why the change from the previous implementation?

Reply:
The previous implementation didn't support external links, like to github.com - BB would click the first result suggestem by AEM, so a page within the AEM instance instead of github.com